src/android/src/main/java/com/reactnativecommunity/slider/ReactSliderManager.java

```diff
@@ -149,6 +149,16 @@ public void setValue(ReactSlider view, double value) {
     view.setOnSeekBarChangeListener(null);
     view.setValue(value);
     view.setOnSeekBarChangeListener(ON_CHANGE_LISTENER);
+    if (view.isAccessibilityFocused() && Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
```

Comment:
DRY. The implementation below is identical to what is already implemented here:
https://github.com/callstack/react-native-slider/blob/269e421bc89137cdb774d0e02c9133e6f208de21/src/android/src/main/java/com/reactnativecommunity/slider/ReactSlider.java#L142
Perhaps we could use that so that the `ReactSlider` will handle all that with his private members?
As an example please take a look at a suggested change below.

Reply:
modified, thank you!